timer.py

```diff
@@ -0,0 +1,16 @@
+import time
+"""
+A function that checks the duration of functions
```

Comment:
Add documentation inside of the function and write is according to the conventions.